ci/jwt/requirements.txt

```diff
@@ -0,0 +1 @@
+jwcrypto
```

Comment:
Should we add `==` or `~=` with version for security reasons?

Reply:
Actually not. It's a test server. I really want to see if tests will break at the same moment as an early feedback from the world outside conjur :-)